JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/GroupBean.java

```diff
@@ -8,132 +11,119 @@
 import java.util.LinkedList;
 import java.util.Map;
 import java.util.HashMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
-import ru.javafiddle.jpa.entity.Access;
-import ru.javafiddle.jpa.entity.UserGroup;
-import ru.javafiddle.jpa.entity.Group;
-import ru.javafiddle.jpa.entity.User;
+import ru.javafiddle.jpa.entity.*;
 
 
 @Stateless
 
 public class GroupBean {
 
-    @PersistenceContext
+    private static final Logger logger =
+            Logger.getLogger(ProjectBean.class.getName());
```

Comment:
What do you mean by that?

23 марта 2016 г., в 1:51, alexander1leonov <notifications@github.com> написал(а):

> In JavaFiddle-ejb/src/main/java/ru/javafiddle/core/ejb/GroupBean.java:
> 
> >  
> >  
> >  @Stateless
> >  
> >  public class GroupBean {
> >  
> > -    @PersistenceContext
> > +    private static final Logger logger =
> > +            Logger.getLogger(ProjectBean.class.getName());
> Typo: should be Logger for GroupBean
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>